ui/lib/pki/addon/components/pki-tidy-form.ts

```diff
@@ -24,6 +24,8 @@ interface Args {
 
 interface PkiTidyTtls {
   intervalDuration: string;
+  minStartupBackoffDuration: string;
+  maxStartupBackoffDuration: string;
```

Comment:
We actually don't need these here. The naming is confusing, but this is for the typescript function below `handleTtl` which is a single input that manages two params, `interval_duration` and `enabled` 

We'd only need to add these if we wanted to do something similar and have these durations attached to a toggle (but I think how you've implemented, without the toggle, it is better)

Reply:
Oops yeah these were me playing around and forgetting to clean these up.

Reply:
Removed in https://github.com/hashicorp/vault/commit/642e5fce92a3058afe0a78127e8b7146b786c74b